src/main/java/controller/Racing.java

```diff
@@ -0,0 +1,78 @@
+package controller;
+
+import controller.RandomNumberGenerator;
+import model.Car;
+import view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Racing {
+    ArrayList<Car> cars = new ArrayList<>();
+    private RandomNumberGenerator randomNumberGenerator = new RandomNumberGenerator();
+    private ResultView resultView = new ResultView();
```

Comment:
불변 가능한 인스턴스 변수는 불변으로 정의하는게 더 안전합니다!
```suggestion
    private final RandomNumberGenerator randomNumberGenerator = new RandomNumberGenerator();
    private final ResultView resultView = new ResultView();
```